Tests/MongoSwiftSyncTests/RetryableWritesTests.swift

```diff
@@ -108,14 +112,19 @@ final class RetryableWritesTests: MongoSwiftTestCase {
                 }
 
                 if test.outcome.error ?? false {
-                    expect(seenError).toNot(beNil(), description: test.description)
```

Comment:
retryable writes tests now have error labels in them and assertions based on their presence, so this runner was updated to do that.